src/components/FlightInfoCard/FlightInfoCard.js

```diff
@@ -0,0 +1,125 @@
+import styled from 'styled-components';
+import { CgArrowLongRight } from 'react-icons/cg';
+
+const FlightInfoCard = ({ flightInfo }) => {
+  const {
+    filghtLogo,
+    flightCompany,
+    flightNumber,
+    departureTime,
+    landingTime,
+    departureCity,
+    flightTime,
+    landingCity,
+    isDiscount,
+    personnel,
+    price,
+    baggage,
+  } = flightInfo;
+  return (
+    <FlightInfoCardWrapper>
```

Comment:
```suggestion
    <Container>
```
저는 그냥 가장 바깥쪽 컴포넌트는 Container로 통일시키는 편입니다

Reply:
오! 팁 감사합니다!